.github/workflows/tests.yml

```diff
@@ -81,6 +85,7 @@ jobs:
           ${{ runner.os }}-${{ matrix.pandas }}-pip-
     - run: |
         pip install --upgrade pip setuptools wheel
+        virtualenv --upgrade-embed-wheels
```

Comment:
The CI is failing because virtualenv is not found, perhaps  try `pip install virtualenv && virtualenv --upgrade-embed-wheels` instead